index.html

```diff
@@ -39,19 +39,42 @@
         <h2>Controls</h2>
         <h3>Date Range</h3>
```

Comment:
Do you think we need a title for Controls and Date Range? I think it looks cleaner without it, and the UI is really intuitive like you have it